### DISCUSSION TITLE
Several instances on a single Raspi

### DISCUSSION THREAD

**Discussion by dirkweisser** [dtcooper/raspotify#659] [Q&A]
Dear all,

My overall plan is to have a single raspi with a single hifiberry soundcard with speakers in different rooms and to "publish" each speaker via raspotify. 
Basic idea was to get the raspi "thinking" that "left" and "right" speaker are different devices. And after this splitting I want to use raspotify to publish each "device" with an own spotify name

I haven't found yet any information (maybe used the wrong keywords) - but is this principally feasible? And if yes - how?

Thanks for any hint in advance

**Comment by Utalor** [comment]
Dear dirkweisser, did you find a solution to your problem ? My raspberry is currently connected to 2 usb DAC and I would like to stream Spotify on both without having to change conf file.

**Comment by kimtore** [comment]
I would try setting up some kind of mapping to create a virtual sound card, preferrably with PipeWire, but Pulseaudio and ALSA should also have these capabilities. Be sure to use mono output to get the same sound everywhere.
